Add docs for email and sms templates for B2B orgs (#5038)
wso2/docs-is: en/includes/guides/branding/customize-email-templates.md

{{email_template_note}}

{% if product_name == "Asgardeo" %}

!!! note "Email templates for B2B applications"

    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/),
    these organizations automatically inherit the email templates of the root organization.

{% elif (product_name == "WSO2 Identity Server" and is_version == "7.0.0") %}

!!! note "Email templates for B2B applications"

    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/),
    you can customize email templates to fit the branding needs of each organization. If you do not customize a
    specific email template, the default email template will be applied.

{% else %}

!!! note "Email templates for B2B applications"

    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/),

>     you can customize email templates to fit the branding needs of each organization. If you do not customize a specific

Reviewer: ```suggestion
    you can customize email templates to fit the branding needs of each organization. If you do not customize an 
```

>     template, the email template will be inherited from the closest ancestor organization with defined email template.

Reviewer: ```suggestion
    email template for an organization, it will inherit the design from the closest ancestor organization with a customized email template.
```

>     If no ancestor has defined email template, the default email template will be applied.

Reviewer: ```suggestion
    If no ancestor has customized the particular email template, the default email template will be applied.
```

{{email_template_note}}

{% if product_name == "Asgardeo" %}

!!! note "Email branding for B2B applications"

>     If you have [organizations]({{base_path}}/guides/organization-management/manage-organizations/) configured,

Reviewer: ```suggestion
    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/), these organizations automatically inherit the branding of the root organization.
```

Author: Fixed with https://github.com/wso2/docs-is/pull/5038/commits/edb4c27ab72cf145858824262f20bf40956ac591

    note that the email branding you configure for your root organization also applies to your organizations.

{% elif (product_name == "WSO2 Identity Server" and is_version == "7.0.0") %}

> !!! note "Email branding for B2B applications"

Author: @AnuradhaSK Any suggestion on the naming convention here. `email branding` or `email templates`

Reviewer 2: email templates is better IMO

Author: Fixed with https://github.com/wso2/docs-is/pull/5038/commits/edb4c27ab72cf145858824262f20bf40956ac591

>     If you have [organizations]({{base_path}}/guides/organization-management/manage-organizations/) configured,

Reviewer: ```suggestion
    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/), you can customize email templates to fit the branding needs of each organization. If you do not customize a specific email template, the default branding will apply.
```

Author: Fixed with https://github.com/wso2/docs-is/pull/5038/commits/edb4c27ab72cf145858824262f20bf40956ac591

    you can configure email branding for each organization. If you have not configured email branding for a given email
    template type and locale in an organization, the default {{product_name}} email branding for the given email
    template type will apply.

{% else %}

!!! note "Email branding for B2B applications"

>     If you have [organizations]({{base_path}}/guides/organization-management/manage-organizations/) configured,

Reviewer: ```suggestion
    If you have set up [organizations]({{base_path}}/guides/organization-management/manage-organizations/) you can customize email templates to fit the branding needs of each organization. If you do not customize a specific template, the branding will be inherited from the closest ancestor with defined branding. If no ancestor has defined branding, the default branding will apply.
```

Author: Fixed with https://github.com/wso2/docs-is/pull/5038/commits/edb4c27ab72cf145858824262f20bf40956ac591

    you can configure email branding for each organization. If you have not configured email branding for a given email
    template type and locale in an organization, the email branding of your immediate parent organization for the given
    email template type and locale will be applied. If your parent organization has no email branding for the given
    email template type and locale, the grandparent organization's email branding for the given email template type and 

>     locale will apply. This will continue all the way until the root organization. If the root organization has no 

Reviewer 2: ```suggestion
    locale will be applied. This will continue all the way until the root organization. If the root organization has no 
```

Author: Fixed with https://github.com/wso2/docs-is/pull/5038/commits/edb4c27ab72cf145858824262f20bf40956ac591

    email branding for the given email template type and locale, the default {{product_name}} email branding for the

>     given email template type will apply.

Reviewer 2: ```suggestion
    given email template type will be applied.
```

Author: Fixed with https://github.com/wso2/docs-is/pull/5038/commits/edb4c27ab72cf145858824262f20bf40956ac591
